mlir/include/mlir/IR/OpAsmInterface.td

```diff
@@ -23,7 +23,7 @@ include "mlir/IR/OpBase.td"
 
 def OpAsmOpInterface : OpInterface<"OpAsmOpInterface"> {
   let description = [{
-    This interface provides hooks to interact with the AsmPrinter and AsmParser
+    This op interface provides hooks to interact with the AsmPrinter and AsmParser
```

Comment:
I don't think op/type/etc. are necessary in the description (the `Op` in the name of the interface already sets the stage for this context).